packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -71,12 +73,18 @@ class CupertinoActivityIndicator extends StatefulWidget {
   /// Defaults to 1.0. Must be between 0.0 and 1.0 inclusive, and cannot be null.
   final double progress;
 
+  /// Whether to use the new style that introduced in iOS14.
+  ///
+  /// This flag will be disabled by default until the official release of iOS 14.
+  final bool useNewStyle;
```

Comment:
I feel like calling it `useNewStyle` isn't very future proof... eg. what will happen when Apple changes it again in iOS 15?

Perhaps a better solution might be to create an enum something like:
```
enum ActivityIndicatorVariant {
  iOS13AndEarlier,
  iOS14,
}
```
I'm not super happy with the naming above, but hopefully the intent is clear.

And whatever naming convention we go with, there's a bunch of other cases where code introduced in this PR talks about the "new" style that should be updated to use the new naming (eg. the names of the tests, the array of alpha values, etc)

Reply:
It would probably also be worth adding a comment to each enum case to describe the specific behaviour

Reply:
I've added the enum `CupertinoActivityIndicatorStyle` to control the style.
The default style still remain iOS13 until the official release of iOS14.